okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -181,16 +181,6 @@ public long read() {
     }
   }
 
-  /** Final test checks and clean up. */
-  @After
-  public void tearDown() throws Exception {
```

Comment:
For verifying tests, I can find the new way to be more complex. Having it in the `@After` means that a test can't forget to have the check and reduces boilerplate just like `@Before`. I only do that when there are at most only a few obviously-special outliers that don't benefit from the behavior, since it is true you don't want it to get out-of-hand. Having the explicit `shutdownAndVerify()` I've found doesn't help much because it is just noise and I wouldn't notice if it was a different-but-slightly-differently-named method. Also, such methods tend to be very generically named, such that I don't really know what they do (shutdown what? verify what?). In this case, having the null check seems very natural/unsurprising/uncomplex and the one test case that doesn't need shutting down is obvious and an outlier (at ~1/45).

In either case, `executor.shutdown();` should probably stay in an `@After`.